doc/interop-test-descriptions.md

```diff
@@ -91,6 +91,87 @@ Client asserts:
 * clients are free to assert that the response payload body contents are zero
   and comparing the entire response message against a golden response
 
+### large_compressed_unary
+
+This test verifies compressed unary calls succeed in sending messages. It
+sends one unary request for every combination of compression algorithm and
+payload type.
```

Comment:
Why does it need to try the different payload types? If it works for one, why wouldn't it work for the others?

The reason the payload type exists is because this service was originally meant for performance testing, so it would want to trigger different on-the-wire sizes in different situations. It doesn't seem useful here.